index.html

```diff
@@ -0,0 +1,187 @@
+                <h3 class="card-subject">
+                    Analysis
+                </h3>
+                <p class="card-description">
+                    Read other user's and contribute with your own interpretation of a given song.
+                </p>
+            </div>
+            <div class="content-card">
+                <img src="./images/icons/forum.png" alt="songwriter logo" class="card-logo">
+                <h3 class="card-subject">
+                    Songwriter
+                </h3>
+                <p class="card-description">
+                    Understand a little more about who wrote your favorite songs. Sometimes they are not who you expect!
+                </p>
+            </div>
+            <div class="content-card">
+                <img src="./images/icons/workshop.png" alt="history logo" class="card-logo">
+                <h3 class="card-subject">
+                    History
+                </h3>
+                <p class="card-description">
+                    No word was ever written outside of a historic context, so we analyse that too!
+                </p>
+            </div>
+            <div class="content-card">
+                <img src="./images/icons/ignite.png" alt="share with us logo" class="card-logo">
+                <h3 class="card-subject">
+                    Share your own
+                </h3>
+                <p class="card-description">
+                    Do you have a beautiful song to share? We love beauty and songs, so please feel free!
+                </p>
+            </div>
+        </div>
+        <button class="content-join">
+            Join our community!
+        </button>
+        <a href="#" class="content-program-dsk">See the whole program</a>
+    </section>
+    <section class="featured">
+        <h2 class="featured-title">
+            Featured Songs
+        </h2>
+        <div id="songs"></div>
+        <button class="featured-button">MORE </button>
+    </section>
+    <section class="main-partners">
+        <p class="partners-title">
+            Partner
+        </p>
+        <div class="partners-above">
+            <img src="./images/partners/mozilla.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/Google.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/navar.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/daumkakao.svg" class="partner-logo-dsk" alt=" logo">
+            <img src="./images/partners/airbnb.png" class="partner-logo-dsk" alt=" logo">
+        </div>
+        <div class="partners-under">
+            <img src="./images/partners/daumkakao.svg" class="partner-logo" alt=" logo">
+            <img src="./images/partners/airbnb.png" class="partner-logo" alt=" logo">
+        </div>
+        </section>
+        <section class="main-footer">
+            <div class="footer-left">
```

Comment:
- [ ] Why don't you use `footer` tag here instead of `section` tag as this is clearly a footer and should have the `footer` tag. This helps to improve the HTML structure of your page and makes your code more [semantic.](https://www.w3schools.com/html/html5_semantic_elements.asp)

Reply:
I did not know this tag existed. I will change it now. Thank you! :)